docker-compose.yml

```diff
@@ -364,6 +364,23 @@ services:
         bahmni:
           aliases:
             - odoo-connect
+  openelis:
+    build: openelis/
+    depends_on:
+      - postgresql
+    environment:
+      OPENELIS_DB_HOST: "${PGSQL_HOST:-postgresql}"
```

Comment:
This can be hardcoded. The rule we have followed until now is that the services names are not flexible.